contracts/contracts/vault/OETHVaultCore.sol

```diff
@@ -269,10 +265,7 @@ contract OETHVaultCore is VaultCore {
             withdrawalRequests[_requestId].queued >
             withdrawalQueueMetadata.claimable
         ) {
-            // Stream any harvested rewards (WETH) that are available to the Vault
-            IDripper(dripper).collect();
-
-            // Add any WETH from the Dripper to the withdrawal queue
+            // Add any WETH to the withdrawal queue
             _addWithdrawalQueueLiquidity();
```

Comment:
`_addWithdrawalQueueLiquidity` can also be deleted since we aren't getting any more WETH. 
I think the entire if statement to check for more liquidity can be removed.

Reply:
We aren't getting any WETH from external Dripper but the Vault might still have unrealized yields in the form of WETH, so that can be accounted towards withdrawal queue shortfall. So, I think we can leave this line (`_addWithdrawalQueueLiquidity`) as it is.

Let's say we have 500 WETH shortfall. And we fund the Vault with 500 WETH. Unless someone explicitly calls the external `addWithdrawalQueueLiquidity()` method, the `_claimWithdrawal` internal method will revert since the metadata isn't updated. This isn't a big deal since we can always call the public method whenever we fund the Vault, but it'd be a minor inconvenience if we remove this

Reply:
> Automatically calling _addWithdrawalQueueLiquidity might prevent us from depositing the desired amount into the strategy B

That makes sense. I don't feel strongly about any of our options. So, I'm fine with whatever you guys decide